src/plugin.ts

```diff
@@ -68,8 +74,9 @@ export interface AbstractPosition {
   address: string // Example: 0x...
   network: string // Example: celo
   appId: string // Example: ubeswap
-  label: string // Example: Pool
+  appName: string // Example: Ubeswap
   tokens: Token[]
+  displayProps: DisplayProps
```

Comment:
For `AbstractPositionDefinition` I think it was good to group the properties used for displaying.

Here in the final Position` representation, I debated whether to put the props at the root level or still grouped under `displayProps`.
For now I decided to leave them grouped so the mapping is clear between with the definition.
But let me know if you have a strong opinion.